src/containers/UserLists/container.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+import { getUserLists as getUserListsAction } from 'Store/userMovieLists/actions'
+import {
+  userListsLoadingSelector,
+  userListsPageSelector,
+  userListsErrorSelector,
+  userListsSelector,
+  isBlankSelector
+} from 'Store/userMovieLists/selectors'
+import UserListsComponent from './component'
+
+class UserLists extends Component {
+  state = {
+    createListModalOpen: false
+  }
+
+  componentDidMount() {
+    const { getUserLists } = this.props
+    getUserLists()
+  }
+
+  handleYourFunctionName = () => {
```

Comment:
`handleYourFunctionName` is just an example

Reply:
✅